index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+   <div class="page-wrapper">
+     <section class="head-top">
+       <div class="header-log">
+        <h2>My Logo</h2>
+       </div> 
+       <nav class="main-nav">
+        <a href="#">☰</a>
+        <ul class="nav-wrapper">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+       </nav>
+     </section> 
+     
+     <section class="headline">
+        <div class="intro-phrase">
+            <h1>I'm Frempong<br>Glad to see <br> you!</h1>
+        </div>
+        <div class="intro-paragraph">
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <div class="connect-wrapper">
+            <h3>LET'S CONNECT</h3>
+            <div class="social-wrapper">
+              <ul class="social-list-wrapper">
+                <li>
+                  <a href="#">
+                    <img src="assets/images/twitter-icon.png" alt="twitter-icon">
+                  </a>
```

Comment:
It isn't a good practice to leave empty `a` references. To fix this, please make sure to add the link to your social media 😄. This will improve accessibility and allow anyone to check your profiles easily 😉.

Reply:
Added all Social Handles.